ksp_plugin/interface_part.cpp

```diff
@@ -41,13 +41,15 @@ void __cdecl principia__PartIncrementIntrinsicForceWithPosition(
     Plugin* const plugin,
     PartId const part_id,
     XYZ const force_in_kilonewtons,
-    XYZ const position) {
+    XYZ const position,
+    XYZ const part_position) {
   journal::Method<journal::PartIncrementIntrinsicForceWithPosition> m(
```

Comment:
Wondering if this method should be called `...AtPosition`.

Reply:
and `Apply` instead of `Increment`.